metaphor/redshift/lineage/extractor.py

```diff
@@ -146,7 +146,11 @@ async def _fetch_lineage(self, sql, conn, db):
         for row in results:
             source_table_name = f"{db}.{row['source_schema']}.{row['source_table']}"
             target_table_name = f"{db}.{row['target_schema']}.{row['target_table']}"
-            query = row["querytxt"].rstrip() if "querytxt" in row else None
+            query = (
+                row["querytxt"].rstrip()
+                if "querytxt" in row and type(row["querytxt"]) == str
```

Comment:
I'd imagine the only possibility for "querytext" is either str or None? 

```suggestion
                row.get("querytxt") is not None
```